load.php

```diff
@@ -15,19 +15,31 @@
 add_action( 'altis.modules.init', function () {
 	$is_cloud = in_array( get_environment_architecture(), [ 'ec2', 'ecs' ], true );
 	$default_settings = [
-		'enabled'                  => true,
-		'cavalcade'                => true,
-		's3-uploads'               => true,
-		'aws-ses-wp-mail'          => $is_cloud,
-		'batcache'                 => $is_cloud,
-		'redis'                    => true,
-		'memcached'                => get_environment_architecture() === 'ec2',
-		'ludicrousdb'              => true,
-		'healthcheck'              => true,
-		'xray'                     => $is_cloud,
-		'email-from-address'       => 'no-reply@humanmade.com',
-		'audit-log-to-cloudwatch'  => $is_cloud,
+		'enabled' => true,
+		'cavalcade' => true,
+		's3-uploads' => true,
+		'aws-ses-wp-mail' => $is_cloud,
+		'batcache' => $is_cloud,
+		'redis' => true,
+		'memcached' => get_environment_architecture() === 'ec2',
+		'ludicrousdb' => true,
+		'healthcheck' => true,
+		'xray' => $is_cloud,
+		'email-from-address' => 'no-reply@humanmade.com',
+		'audit-log-to-cloudwatch' => $is_cloud,
 		'php-errors-to-cloudwatch' => $is_cloud,
+		'cache' => [
```

Comment:
I think this should be called `page-cache` maybe?

Reply:
Yeah makes sense to be more specific 👍